mysql-test/suite/galera/t/galera_gcache_recover_manytrx.test

```diff
@@ -128,6 +128,9 @@ DELIMITER ;|
 --connection node_2
 SET SESSION wsrep_sync_wait = 0;
 
+# we need to give it a chance to put something in gcache and in case of encryption for some enc cache pages to be flushed
+--sleep 18
```

Comment:
Will this also work on jenkins, isn't there a better check?

Reply:
This magic --sleep 18 is here a few lines below in the original test.
The flow of the original test is:
1. Start transactions on node_1
2. Wait on node_2 just for CREATE TABLE to be replicated (and probably some inserts, but do not care how much of them)
3. kill node_2
4. wait for 18 secs. node_1 still processes inserts
5. start node_2 - it should do gcache recovery and join with SST.

There is a problem with points 2 and 3 when we use gcache encryption. As encryption uses in-memory cache which is being flushed only when there are no more cache pages available, if we kill immediately in point 3, there is nothing in GCache file. So the recovery will not take place.
We need to wait for some time for GCache encryption cache to be flushed internally (and this will be done if we receive more transactions from node_1). Then, after node_2 restart it will see that there is "something" in gcache file and the recovery will take place.
Why 18 secs? well, because the original test used 18 as well :).  There is no way to say that the encryption cache has been flushed (the only way is to examine gcache file itself, but we still would need to wait for some arbitrary time, check and then wait again if check fails...) but I didn't want to change this test too much.